src/apps/user_auth/models.py

```diff
@@ -0,0 +1,66 @@
+from django.utils import timezone
+
+from django.contrib.auth.base_user import AbstractBaseUser, BaseUserManager
+from django.contrib.auth.models import PermissionsMixin
+from django.db import models
+
+
+class UserManager(BaseUserManager):
```

Comment:
Let's get rid of this model manager? Should we keep it around instead of just using functions?

Reply:
How would we do this with functions vs a model manager?